src/pages/kyoungjinLee/Login/Login.js

```diff
@@ -1,8 +1,76 @@
-import React from 'react';
+import { Link, useNavigate } from 'react-router-dom';
+import { useState } from 'react';
 import './Login.scss';
+import '../../../styles/common.scss';
+
+function Footer() {
+  return (
+    <footer>
+      <Link to="/main-kyoungjin" class="link">
+        비밀번호를 잊으셨나요?
+      </Link>
+    </footer>
+  );
+}
 
 const LoginKyoungjin = () => {
-  return <div>Login</div>;
+  const [IdData, setIdData] = useState('');
```

Comment:
```suggestion
  const [idData, setIdData] = useState('');
```
state의 네이밍 컨벤션은 camelCase입니다.